### DISCUSSION TITLE
2021.06.15

### DISCUSSION THREAD

**Discussion by Settpark** [malaheaven/issue-tracker#84] [General]
iOS: 
- 금일 네트워크 관련 테스트 코드 작성했습니다.
- 네트워크의 역할 분리 했습니다. 
- OAuth 구현하고 해당 키 값 저장하는 방법 찾아봤습니다. (키체인으로 저장하는 방식)
**iOS는 오늘 행복합니당** 😀

**Comment by malaheaven** [comment]
### BE
도메인 개발
- jpa 삽질 😢 (내일 해결 예정)

BE도 오늘 행복합니다

**Comment by 17-sss** [comment]
**FE**

- Modal 켜고 끄는게 왜 어렵죠..?
  - 끝날때까지 난 어디로.. (항상 산으로 가버리는 프로 등산러 라노)
- OAuth 적용이 거의 끝나갑니다 (json & Robin 멋집니당!)

**_FE도 오늘 행복합니다._**
